python/lsst/ts/xml/testutils.py

```diff
@@ -59,6 +59,11 @@
     'VALUETYPE', 'VOID', 'WCHAR', 'WSTRING'
 ]
 
+"""Define the list of IDL Types"""
+idl_types = ['boolean', 'byte', 'octet', 'short', 'int', 'long', 'long long',
+             'unsigned short', 'unsigned int', 'unsigned long',
+             'unsigned long long', 'float', 'double', 'string', 'char']
+
```

Comment:
For future reference: the "in" test would be more efficient if `idl_types` was a set, e.g. `idl_types = set(['boolean', ...])`

Efficiency is no big deal here as I imagine the test runs fairly quickly anyway.

Reply:
Thanks for educating me!